### PR TITLE
HCP Form Address Autocomplete

### PR DESCRIPTION
**Pull request by jamesjypark**
This PR added address autocomplete on HCP Form.

Note that this change requires `API_KEY` in `request.html` to be replaced with the correct GCP API key.
Please contact me or Jason to acquire the API key.

Below is the screenshot of the UI change
Screen Shot 2020-04-15 at 7 39 56 PM

### CODE REVIEW

**Comment by wsonnex** [comment]
Blocked on reviewing this until the previous PR (https://github.com/bccovid-19/track/pull/9) is merged. Perhaps unavoidable but in general be careful about mixing PRs like this.

**Comment by wsonnex** [comment]
@jamesjypark sorry this fell behind but I'm now getting merge conflicts trying to get `develop` merged into this branch. Could you try to resolve them?

**Comment by jamesjypark** [comment]
@wsonnex Yup, I'll look into it first thing tomorrow!

File: templates/apps/bcc3d.ca/_config.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ description: >- # this means to ignore newlines until "baseurl:"
 
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "{{ scheme }}://{{ apps.bcc3d_ca.hostname }}"
+google_maps_api_key:
```
Comment (wsonnex) [inline templates/apps/bcc3d.ca/_config.yml:34]: Does anyone have an API key we can use? (@chaosaffe @jamesjypark)

**Pull request merged**